templates/core/terraform/main.tf

```diff
@@ -99,6 +99,7 @@ module "firewall" {
   location                   = var.location
   resource_group_name        = azurerm_resource_group.core.name
   log_analytics_workspace_id = module.azure_monitor.log_analytics_workspace_id
+  debug                      = var.debug
```

Comment:
I'm not sure I'd have picked "debug" as it as an implication that we're also setting this deployment up to have additional logging, and be easier to debug. I'm not sure that's true.

I'd have done with "development" rather than "debug" but this is fine. Naming stuff is hard :)

Reply:
I understand what you mean... this wasn't my choice and it's being using in a few other places to have TF implement other behaviors - like in the KV IIRC.